thoth/slo_reporter/sli_apis/sli_user_api.py

```diff
@@ -32,6 +32,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+_DRY_RUN = bool(int(os.getenv("DRY_RUN", 0)))
```

Comment:
this is already available in `configuration.dry_run`

Reply:
@pacospace I deleted `self.ceph_sli = "dry_run"` in `Configuration` when `dry_run` is `True`, just checking whether it is `True` in each `sli context` should be enough, WDYT?